src/App.js

```diff
@@ -1,10 +1,68 @@
 import React from 'react';
 import './App.css';
+import Button from "./components/Button/Button";
+import handbag1 from './assets/bag_1.png';
+import handbag2 from './assets/bag_2.png';
+import handbag3 from './assets/bag_3.png';
+import handbag4 from './assets/bag_4.png';
+import brandimage from './assets/brand.png';
+import designersimage from './assets/our_story.png';
+import Article from "./components/Article/Article";
+import Tile from "./components/Tile/Tile";
+
 
 function App() {
-  return (
-      <h1>Handbags & Purses</h1>
-  );
+    return (
+        <>
+            <h1>Handbags & Purses</h1>
+            <nav>
+                <Button buttonType="button" isDisabled={false}
+                        buttonClickHandler={() => console.log("To the collection!")}>To the collection</Button>
+                <Button buttonType="button" isDisabled={false} buttonClickHandler={() => console.log("Shop all bags")}>Shop
+                    all bags</Button>
+                <Button buttonType="button" isDisabled={true}
+                        buttonClickHandler={() => console.log("Pre-orders")}>Pre-orders</Button>
+            </nav>
+            <main>
+                <Article
+                    tag="Best Seller"
+                    image={handbag1}
+                    subTitle="The handy bag"
+                    price="€400,-"
```

Comment:
Volgens mij haal je nu de prijs niet uit het component., maar is deze hard-coded. Wat je had kunnen doen (en zo staat het ook in de uitwerkingen) is dit: <h4>€{price},-</h4>